mmv1/products/dlp/DiscoveryConfig.yaml

```diff
@@ -125,6 +125,21 @@ properties:
           - name: 'folderId'
             type: String
             description: The ID for the folder within an organization to scan
+  - name: 'otherCloudStartingLocation'
+    type: NestedObject
+    properties:
+      - name: 'awsLocation'
+        type: NestedObject
+        properties:
+          - name: 'accountId'
```

Comment:
According to [the docs](https://cloud.google.com/sensitive-data-protection/docs/reference/rest/v2/OtherCloudDiscoveryStartingLocation#awsdiscoverystartinglocation) it's possible to specify `accountId` or `allAssetInventoryAssets`, but not both. I'm not sure if leaving both unset is valid, but in either case you can improve the user experience by adding either:

- [conflicts](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#conflicts) to both fields, if they're allowed to be unset
- [exactly_one_of](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of) if one of them is required to be set